### ISSUE TITLE
Get participants from the call

### ISSUE THREAD

**Issue by bijaykush** [microsoftgraph/microsoft-graph-comms-samples#140]
**First case**
I called bot using my Teams account. The call was answered by using an answer API and then invited a participant. Now there are 3 participants: bot, my Teams account and invited participant. 
Now when I try to get the participants of the call, it gives no participants. 

**Second case**
When a group call is initiated at once i.e calling bot and participants at a time, I can easily get the participants in the call.

Does it mean that we can only get participants of the call if the call generated at once with multiple participants?

**Comment by ksikorsk** [comment]
The first case is an "escalation" scenario (escalate a 1:1 call to a multiparty call), which is not yet supported.

One work around is to create an new ad-hoc meeting, and invite both participants to the meeting.  You can invite the original caller and specify the `ReplacesCallId` as part of `InvitationParticipantInfo`, which will pull them from the original call and move them to the new meeting.  You may have to send 2 `invite` requests (as opposed to a single request with 2 `invitationParticipantInfo` objects).

**Comment by bijaykush** [comment]
@ksikorsk 
Thanks for the fruitful response. I will look at the workaround process suggested by you.

But still, I want to know if there any possibility of supporting the **first case** directly in future By MS Teams?

**Comment by ksikorsk** [comment]
Yes, supporting escalation is part of our plan.  @ananmishr for timeline.

**Comment by surezg** [comment]
@ksikorsk Any updates on the timeline please? I am also in the same situation like @bijaykush. This is very critical for help desk scenarios.

**Comment by ananmishr** [comment]
Hi @surezg and @bijaykush the timeline for the escalation scenario may be sometime during the second half of this year

**Comment by surezg** [comment]
thanks @ananmishr 

@ksikorsk I have tried the workaround you have recommended.  It works when the calls originates from teams user. I have tried calling from PSTN number where it lands to bot through a service account user as recommended by microsoft. In this case, how can I invite the PSTN caller in to the meeting? For teams user, I can use the Object ID.  For PSTN caller, I have tried using the "service account" user tenantId which returns "Request authorization tenant mismatch.".

Any help on this please?

**Comment by gence** [comment]
If the order is as follows, would it work?

1. 1-1 audio meeting between two human
2. Add bot as 3rd participant

Or would this still end up not showing participant info?

**Comment by surezg** [comment]
@gence  I am working on help desk scenario where customer (from teams/PSTN) calls bot to land in IVR, then customer pick up the ivr menu, then bot will add available agent in to the existing call. this is working as I want. Only thing is SDK is not throwing back the participants if any of the participant is dropped. If 2 participants are dropped, then the call gets terminated by itself. Any help on this please?

The second workaround suggested by @ksikorsk  doesn't work for PSTN callers.

@ananmishr , @ksikorsk  - any help on this please. This is very critical blocker.

**Comment by ananmishr** [comment]
@xixian73 for help with PSTN

**Comment by surezg** [comment]
> @xixian73 for help with PSTN

@ananmishr  - Can federated user call the Bot from teams?

**Comment by daltskin** [comment]
> ne work around is to create an new ad-hoc meeting, and invite both participants to

In the escalation scenario, if creating a new ad-hoc meeting, can you move over all the existing participants server-side - without each user having to do anything?
